lib/wallets/adapters/TonWalletAdapter.ts

```diff
@@ -0,0 +1,65 @@
+import { Cell } from "ton";
```

Comment:
nit: is TonWallet always a ton chrome extension? maybe change the names of the classes to reflect that, because (at least to me) it's confusing to read it and have to think whether it refers to tonhub or ton chrome ext, the name is a bit generic.